cel/cel_test.go

```diff
@@ -45,7 +44,13 @@ func Example() {
 		// Function to generate a greeting from one person to another.
 		//    i.greet(you)
 		decls.NewFunction("greet",
-			decls.NewInstanceOverload("greet_string_string",
+			decls.NewInstanceOverload("greet_i_string_string",
+				[]*exprpb.Type{decls.String, decls.String},
+				decls.String)),
+		// Function to generate shake_hands between two people.
```

Comment:
Actually, would you mind creating a separate example for Instance vs. Global overload?